### ISSUE TITLE
Installation von German Setup in einen MRG-verseuchten Shop

### ISSUE THREAD

**Issue by kurtdeserter** [firegento/firegento-germansetup#198]
Hallo!
Gibt es eigentlich eine Erklärung, wie man German Setup in einem Shop zum Laufen bekommt, der zuvor mit Market Ready German verseucht wurde?
Wir haben magento 1.8 frisch installiert, einen dump der Artikeln, Attribut Sets und Kategorien gemacht, im 1.8 eingespielt und bekommen immer folgende Fehlermeldung, wenn wir German Setup installieren wollen:

Exception during cache and session cleaning: Error in file: "/home/www/p207962/html/magento/app/code/community/FireGento/GermanSetup/sql/germansetup_setup/mysql4-install-0.5.0.php" - SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '63-General' for key 'UNQ_EAV_ATTRIBUTE_GROUP_ATTRIBUTE_SET_ID_ATTRIBUTE_GROUP_NAME'
Bislang sind alle Lösungsansätze gescheitert.

Danke

**Comment by mklooss** [comment]
Entweder du löschst das Attribute "delivery_time" oder du kommentierst 
es im installer aus...

**Comment by kurtdeserter** [comment]
Hallo!

Danke für den Tipp, hab das Attribut gelöscht, aber das Problem bleibt bestehen!
Es ist ja auch eigentlich nicht "delivery time" welches Fehler macht, oder lese ich die Fehlermeldung falsch?

Am 04.11.2013 um 11:21 schrieb Mathis Klooß notifications@github.com:

**Comment by mklooss** [comment]
sicher dass es die gleiche Fehlermeldung ist,
weil im Installer wird nur das Attribute delivery_time erstellt:
https://github.com/firegento/firegento-germansetup/blob/master/src/app/code/community/FireGento/GermanSetup/sql/germansetup_setup/mysql4-install-0.5.0.php

/home/www/p207962/html/magento/app/code/community/FireGento/GermanSetup/sql/germansetup_setup/mysql4-install-0.5.0.php

**Comment by kurtdeserter** [comment]
Ja, macht mich stutzig, hab ich mir auch schon angesehen, aber die Fehlermeldung ist korrekt angeführt.
Wie kann man denn am besten die Extension manuell installieren. 
Weiß nicht, wie ich delivery time in der php Datei auskommentieren kann, bevor ich die Ext. installiere.

**Comment by riker09** [comment]
Zieh' die Moduldateien entweder über Magentocommerce.com oder direkt über GitHub. Entpack' das Archiv und bearbeite die PHP-Datei.

**Comment by kurtdeserter** [comment]
Auf die Gefahr hinauf, dass ich mich blamiere….hab das Archiv runtergeladen, entpackt und am Server kopiert, die PHP-Datei bearbeitet.
Nun weiß ich aber nicht, wie ich die Extension zum Laufen bekomme

Am 04.11.2013 um 12:43 schrieb Volker Thiel notifications@github.com:

**Comment by mklooss** [comment]
nicht falsch verstehen, aber auch mich wirkt es so als ob du dir erstmal die PHP Basics anschauen solltest statt Magento. mit Auskommentieren musst du in der Datei "app/code/community/FireGento/GermanSetup/sql/germansetup_setup/mysql4-install-0.5.0.php" im Magento Ordner

Die Fehlermeldung sagt, dass er versucht eine Attribute Gruppe zu erstellen, es aber nicht darf, da es schon eine mit dem Namen gibt.

http://php.net/manual/en/language.basic-syntax.comments.php

**Comment by kurtdeserter** [comment]
Ich verstehe das nicht falsch, hab auch nie gefragt was auskommentieren bedeutet.
Die Frage war sogar nur Extension-spezifisch und nicht mal auf Magento bezogen!?
Ich verstehe sogar die Fehlermeldung!

Interessant finde ich, dass gerade eine Gruppe, die darauf aus ist, dass sie German Setup vertreibt (und da werden wohl nicht nur Neuinstallationen die Zielgruppe sein!?) diese Eigenart der Installation kalt lässt.

Habe es aber mittlerweile gelöst.

Danke trotzdem

Am 05.11.2013 um 11:15 schrieb Mathis Klooß notifications@github.com:

**Comment by mklooss** [comment]
nirgends ist beschrieben dass es ein DropIn Replacement ist... Da MRG auch immer weiter entickelt wird kann man hier nicht alles abfangen.
